src/cdk/v1/heap/mapping/track.yaml

```diff
@@ -18,8 +18,9 @@
     type: flatJson
 
 - destKey: timestamp
-  sourceKeys: timestamp
```

Comment:
though the preference here is `timestamp`, how come it's not being respected ?
can you describe what was the bug ?

Reply:
It was the order problem in `rudder-transformer-cdk`.

Reply:
```
"timestamp": ["originalTimestamp", "timestamp"],
```
Currently this is how the order looks in `rudder-transformer-cdk` and the generic-mapping is being referred from `rudder-transformer-cdk`